### PR TITLE
Fix font & improve ui

### PR DESCRIPTION
**Pull request by nathangobinet**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -3,5 +3,4 @@ html {
 	background-attachment: fixed;
 }
 
-@import '~typeface-roboto/index.css';
-@import '~typeface-caveat/index.css';
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Kanit:wght@400;700&family=Open+Sans&family=Ubuntu+Mono&display=swap');
```
Comment (adrien-nf) [inline src/index.css:6]: Should we install the font locally instead?

Comment (nathangobinet) [reply src/index.css:6]: Read https://web.dev/font-best-practices/#using-self-hosted-fonts

Not sure if we really care

Comment (adrien-nf) [reply src/index.css:6]: Good for me.

**Review by adrien-nf — APPROVED** [review]

**Review by adrien-nf — APPROVED** [review]

**Comment by nathangobinet** [comment]
Merged

**Pull request closed without merging**